src/generator/src/cmd/generate.ts

```diff
@@ -189,7 +192,7 @@ ${yaml.dump({ 'input-file': filesByTag[tag] }, { lineWidth: 1000})}
   await writeFile(bicepReadmePath, generatedContent);
 }
 
-async function generateSchema(logger: ILogger, readme: string, outputBaseDir: string, logLevel: string, waitForDebugger: boolean) {
+async function runAutorest(logger: ILogger, readme: string, outputBaseDir: string, logLevel: string, waitForDebugger: boolean) {
```

Comment:
Would users rely on this file to generate schemas? Unlike with the version in `src/autorest.bicep/test/integration/utils.ts`, I don't see a way to run autorest with the flag `--arm-schema=true` via this command.

Reply:
To start off with, I'm planning to add this as a git submodule to https://github.com/Azure/azure-resource-manager-schemas, and running it directly from the orchestration scripts there.

It has its own schemas folder structure and fairly complex generation script that I'm not ready to change yet. If I'm able to clean that up sufficiently, I might try and migrate it into this repo.